src/app/home/info/info.component.html

```
@@ -1,149 +1,57 @@
-<!-- Masthead -->
-<div class="background relative">
+<div class="background console-route relative">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)